### DISCUSSION TITLE
How do we know what is supported and what not?

### DISCUSSION THREAD

**Discussion by oorestisime** [oven-sh/bun#1781] [General]
I see in releases that some packages are supported and some not  etc but how do we get an overview?

* express is supported now? (i saw 0.1.7 that it was partially supported).
* is any db work supported now? i.e we are using knex.js and postgres driver

Generally speaking looking to see if we can use this in our project. I guess it is still not production ready but really looking to understand what we might be missing from getting there.

**Comment by zypher420** [comment]
in this main repo `oven-sh/bun` is a folder `packages` inside there is a folder called `bun-types`.
